src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -5092,6 +5090,21 @@ namespace LowTempRadiantSystem {
         }
     }
 
+    Real64 RadiantSystemBaseData::calculateOperationalFraction(Real64 const offTemperature, Real64 const controlTemperature, Real64 const throttlingRange)
+    {
+        Real64 temperatureDifference = std::abs(offTemperature - controlTemperature);
+        if (temperatureDifference <= 0.0) {
+            return 0.0; // No temperature difference--turn things off (set to zero); technically shouldn't happen
+        } else if (throttlingRange < 0.001) {
+            return 1.0; // Throttling range is essentially zero and there is a temperature difference--turn it full on
+        } else if (temperatureDifference >= throttlingRange) {
+            return 1.0; // Throttling range non-zero and temperature difference greater than the throttling range--turn it full on
+        } else {
+            return temperatureDifference/throttlingRange;   // Temperature difference is non-zero and less than the throttling range--calculate the operation fraction
```

Comment:
There should be a space before and after the division operator.

Reply:
Will fix this when we resolve the minimum throttling range issue.